cernopendata/modules/fixtures/data/records/atlas-ATL-PHYS-PUB-2022-039.json

```diff
@@ -0,0 +1,76 @@
+[
+  {
+    "abstract": {
+      "description": "<p>FIXME This dataset serves for performance studies of constituent-based jet taggers on large radius boosted top quark jets reconstructed from optimized jet input objects in simulated collisions at √ 𝑠 = 13 TeV. Several taggers which consider all the information contained in the kinematic information of the jet constituents are tested, and compared to a tagger which relies on high-level summary quantities similar to the taggers used by ATLAS in Runs 1 and 2. Several constituent based taggers are found to out-perform the high level quantity based tagger, with the best achieving a factor of two increase in background rejection across the kinematic range.</p> <p>FIXME This dataset consists of two files, the training data and the test data, both available in H5 format. The dataset semantics and code usage examples are linked below.</p>"
```

Comment:
```suggestion
      "description": "<p> Boosted top tagging is an essential binary classification task for experiments at the Large Hadron Collider (LHC) to measure the properties of the top quark. The ATLAS Top Tagging Open Data Set is a publicly available data set for the development of Machine Learning (ML) based boosted top tagging algorithms. The data are split into two orthogonal sets, named train and test and stored in the HDF5 file format, containing 42 million and 2.5 million jets respectively. Both sets are composed of equal parts signal (jets initiated by a boosted top quark) and background (jets initiated by light quarks or gluons). For each jet, the data set contains:
- The four vectors of constituent particles
- 15 high level summary quantities evaluated on the jet
- The four vector of the whole jet
- A training weight
- A signal (1) vs background (0) label
There is one rule in using this data set: the contribution to a loss function from any jet should always be weighted by the training weight. Apart from this a model should separate the signal jets from background by whatever means necessary. <\p>"
```

Reply:
Is it possible to include markdown in the .JSON skeleton? Ideally the items after "the data set contains" are formatted as a bulleted list.

Reply:
It's possible to include HTML formatting; I have introduced list.